packages/cockroach/extra/cockroach.sh

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
+
+set -xe
+
+myip=`/opt/mesosphere/bin/detect_ip`
+nodes=`cat /run/dcos/cockroach/nodes`
+
+exec /opt/mesosphere/active/cockroach/bin/cockroach start \
+     --logtostderr \
+     --cache=100MiB \
+     --store=/var/lib/dcos/cockroach \
+     --insecure \
+     --advertise-host=${myip} \
+     --host=${myip} \
+     --port=26257 \
+     --http-host=127.0.0.1 \
+     --http-port=8090 \
+     --log-dir= \
+     --extra-1.0-compatibility \
```

Comment:
Do we need this flag in `1.13`?

Reply:
@gpaul can you help here?

Reply:
Taken care of in this PR, as well as in DC/OS Enterprise.

Removed the flag.